src/examples/upload_package_using_cli.yml

```diff
@@ -0,0 +1,24 @@
+description: >
+  Upload a Java Maven package to Cloudsmith using the Cloudsmith CLI.
+usage:
+  version: 2.1
+  orbs:
+    cloudsmith-maven: ft-circleci-orbs/cloudsmith-maven@0.0.1
+  jobs:
+    build:
+      docker:
+        - image: cimg/openjdk:21.0.2
+      steps:
+        - checkout
+        - run:
+            name: Build maven package
+            command: mvn package
+            working_directory: .circleci/simple-maven-package-test/maven-package-test
+        - cloudsmith-maven/upload_package_using_cli:
+            repository: "financial-times-internal-releases"
+            dist_dir: .circleci/simple-maven-package-test/maven-package-test
```

Comment:
This wdist_dir won't be correct for someone to copy and paste into their own config.yml